src/core/services/theming/theming.js

```diff
@@ -689,6 +717,45 @@ function generateAllThemes($injector, $mdTheming) {
     delete palette.contrastStrongLightColors;
     delete palette.contrastDarkColors;
 
+    function getContrastType(hueName) {
+      if (defaultContrast === 'light') {
+        if (darkColors.indexOf(hueName) > -1) {
```

Comment:
I prefer the `!== -1` check for `indexOf` calls.

See https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf